src/main/java/org/broadinstitute/hellbender/tools/copynumber/PostprocessGermlineCNVCalls.java

```diff
@@ -654,4 +653,25 @@ private static File getCopyNumberSegmentsFile(final File pythonSegmenterOutputPa
         }
         return unsortedIntervalCollectionsFromModels;
     }
+
+    /**
+     * Validate that the union of shard's interval lists does not have singleton intervals, i.e. intervals that
+     * are the only ones on their corresponding contigs.
+     */
+    private void checkForSingletonIntervalAbsence(final List<SimpleIntervalCollection> intervalCollections){
+        final List<SimpleInterval> concatenatedIntervalList = new ArrayList<>();
+        IntStream.range(0, numShards).forEach(i -> {
+            concatenatedIntervalList.addAll(intervalCollections.get(i).getIntervals());
+        });
+
+        final Map<String, Long> contigToCountMap = concatenatedIntervalList.stream()
```

Comment:
I'd make the variable names and code here consistent with the code in FilterIntervals.

Reply:
Okay I did that as much as possible